code/src/index.css

```diff
@@ -1,13 +1,122 @@
 body {
-  margin: 0;
   font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", "Roboto", "Oxygen",
-    "Ubuntu", "Cantarell", "Fira Sans", "Droid Sans", "Helvetica Neue",
-    sans-serif;
+  "Ubuntu", "Cantarell", "Fira Sans", "Droid Sans", "Helvetica Neue",
+  sans-serif;
   -webkit-font-smoothing: antialiased;
   -moz-osx-font-smoothing: grayscale;
+  background-image: url("Assets/footprints.png");
+  background-position: center;
+  background-size: 100%;
+  background-repeat: no-repeat;
 }
 
 code {
   font-family: source-code-pro, Menlo, Monaco, Consolas, "Courier New",
     monospace;
 }
+
+h1 {
+  font-size: 30px;
+  text-align: center;
+}
+
+h2 {
+  font-size: 20px;
+  text-align: center;
+  margin: 20px;
+  font-weight: 500;
+}
+
+p {
+  font-size: 20px;
+  text-align: center;
+}
+
+form {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  padding: 20px;
+}
+
+.name-question {
+  font-size: 20px;
+  text-align: center;
+  margin: 10px;
+}
+
+.name {
+  width: 150px;
+  height: 25px;
+  margin: 10px;
+  border: 2px solid black;
+}
+
+.fur-question {
+  font-size: 20px;
+  text-align: center;
+  margin: 10px;
+  display:flex;
+  flex-direction: column;
+}
+
+.color-question {
+  font-size: 20px;
+  text-align: center;
+  margin: 10px;
+}
+
+.select {
+  width: 150px;
+  height: 30px;
+  margin: 10px;
+  border: 2px solid black;
+}
+
+.email-section {
+  margin: 20px;
+}
+
+.email {
+  width: 170px;
+  height: 25px;
+  margin: 10px;
+  border: 2px solid black;
+}
+
+.submit {
+  border-radius: 40px;
+  border: 2px solid black;
+  padding: 5px;
+  width: 70px;
+  height: 40px;
+  font-size: 20px;
+}
+
+.summary-section {
+  color: black;
+  display: flex;
+  flex-direction: column;
+}
+
+.summary-section h1 {
+  font-size: 20px;
+  font-weight: 800;
+  margin: 30px;
+} 
+
+.greet {
+  display: flex;
+  flex-direction: column;
+}
+
+.summary {
+  display: flex;
+  flex-direction: column;
+  font-style: oblique;
+}
+
+.ending {
+  font-weight: 700;
+  height: 200px;
+}
```

Comment:
Good work with the classNames! It's easy to understand and where they take place. I have no other comments on the CSS.